core/src/test/java/org/fao/geonet/kernel/LocalXLinksUpdateDeleteTest.java

```diff
@@ -90,6 +92,7 @@ public void execute() throws JobExecutionException {
     }
 
     @Test
+    @Ignore(value = "Doesn't run along the others in core")
```

Comment:
The test also comes from geocat.ch maybe @cmangeat could help making it to work with other tests ?